mscore/editdrumset.cpp

```diff
@@ -239,12 +307,25 @@ void EditDrumset::apply()
 //   itemChanged
```

Comment:
Can you fix the comments "itemChanged" and put it before the right method ? Also add comment for new methods please.